.env.template

```diff
@@ -1,6 +1,14 @@
-# Flask server configuration.
+### Flask ###
+
+# Server configuration.
 FLASK_APP=app
```

Comment:
This needs to be updated to app/app now that you've moved app.py into a subdirectory.

Also the `__init__.py` file needs to be moved into the app/ folder.